frontend/src/api/pipelines/custom.ts

```diff
@@ -5,19 +5,19 @@ import {
   GetPipelineAPI,
   DeletePipelineAPI,
   ListPipelinesRunAPI,
-  ListPipelinesRunJobAPI,
+  ListPipelineRecurringRunsAPI,
```

Comment:
I am not sure whether is has to be `ListPipelinesRecurringRunAPI` or is it fine to use `ListPipelineRecurringRunsAPI` here.

Reply:
i think this one is fine